### DISCUSSION TITLE
Security Baselines not showing

### DISCUSSION THREAD

**Discussion by Roland0909** [Micke-K/IntuneManagement#242] [Ideas]
Hej Micke

The Endpoint Security section not showing all Security Baselines.
Only "Defender for Endpoint Baseline Standard" is popping up and I have three more.

Is there a quick fix for it, or how do you see it?

Hälsningar Roland

**Comment by Micke-K** [comment]
Hello,

Check under Settings Catalog. The new versions uses the Settings Catalog API

Cheers!

**Reply by Roland0909** [reply]
OK, thanks - found it.

Mvh

**Reply by saqib-s** [reply]
Thanks Micke-K!,

Any chance this can be broken out into a separate category like "Security Baselines (Setting Cat.)" for viewing purposes?

**Comment by Micke-K** [comment]
Hello,

I'm working on it :). The next major release will have that. I've been too busy so it takes a bit longer than I thought. I am rewriting it from scratch. No clue when I can release it. Far from close at the moment. 

Cheers
